src/pretty.jl

```diff
@@ -347,11 +347,55 @@ end
 p_stringh(style::S, cst::CSTParser.EXPR, s::State) where {S<:AbstractStyle} =
     p_stringh(DefaultStyle(style), cst, s)
 
+
+format_docstrings!(something_else) = nothing
+
+function format_docstrings!(document::MD)
+    for clause in document.content
+        format_docstrings!(clause)
+    end
+    nothing
+end
+
+function format_docstrings!(doctest::Code)
+    language = doctest.language
+    code = doctest.code
+    doctest.code = if startswith(language, "@example") ||
+        startswith(language, "@repl") ||
+        startswith(language, "@eval")
+        format_text(code)
+    elseif startswith(language, "jldoctest")
+        if occursin(r"^julia> "m, code)
+            join(
+                (
+                    string("julia> ", join(
+                        (
+                            line for line in split(format_text(input), '\n')
+                        ),
+                        "\n       "
```

Comment:
btw is the spacing here for `julia>` ?

Reply:
Yeah. All of the code after `julia> ` should be indented by 6 spaces, I think.